worker/uniter/remotestate/watcher_test.go

```diff
@@ -188,9 +188,9 @@ func (s *WatcherSuiteIAAS) TestSnapshot(c *gc.C) {
 	s.signalAll()
 	assertNotifyEvent(c, s.watcher.RemoteStateChanged(), "waiting for remote state change")
 
-	// Note that the configuration change is updated on both application and
-	// charm config which increments it twice.
-	expectedVersion := 3
+	// Note that the configuration change is updated on both application config and
```

Comment:
I think this should be " on both trust changes and address changes "
The uniter watches for trust changes, and on the server side it's done in terms of app config changes.

Reply:
Right, that makes sense.